README.md

```diff
@@ -21,6 +22,9 @@ cd $HOME/jrtorrent
 screen -dmS jrtorrent java -Xmx32m -jar build/libs/jrtorrent-all-0.1.jar
 ```
 
+#### requirements
+* ``Java 1.8``
```

Comment:
can you also change it to 'java 8'?  with java 7 oracle removed prefix '1.' from the release name, although it's still in full build version